opencti-platform/opencti-front/tests_e2e/artifact/createArtifact.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { ArtifactPage} from "../model/Artifact.pageModel";
+import { ArtifactImportPage} from "../model/ArtifactImport.pageModel";
+import { expect, test } from "../fixtures/baseFixtures";
+
+
+test('Validate a new artifact file', async ({ page }) => {
+        const artifactPage = new ArtifactPage(page);
+        const artifactImport = new ArtifactImportPage(page);
+        await page.goto('/dashboard/observations/artifacts')
+        await artifactPage.addNewArtifactImport().click();
+        artifactImport.getFileInput();
+        // await artifactImport.fillFileInput('File');
```

Comment:
To remove? 
And it seems you have indentation issues. Is the linter applied on test files here?

Reply:
Indeed, thanks :)
No issue with the linter

Reply:
After testing, there is an issue as the linter seems not linting e2e spec files but not related to your PR.